### ISSUE TITLE
quention about the multi-label classfication

### ISSUE THREAD

**Issue by diaodeyi** [jackyjsy/SGGAN#5]
Sorry to bother again, as i kown , the ACGAN try to solve the multi-class problem, but the SGGAN need to solve the multi-label problem right? So how can this structure can differentiate different labels of single image without other loss or trick ,but just use a classfication loss in the ACGAN.

**Comment by diaodeyi** [comment]
I mean the binary_cross_entropy_with_logits loss in your code revealing that this mask is not multi-class but multi-label.

**Comment by jackyjsy** [comment]
Yes. The binary_cross_entropy_with_logits loss is for multi-labels (blonde hair, brown hair, black hair, gender and age) that multiple classes may coexist (e.g. a face image can be black hair young male). Then the classification loss on fake images are used in the generator loss (Eq. 10 in [paper](https://arxiv.org/pdf/1901.01569.pdf)). Such loss term guides the generator to generator class specific fake images. Such design is first introduced by [StarGAN](https://arxiv.org/pdf/1711.09020.pdf). Hope I answered your question.

**Comment by diaodeyi** [comment]
Thanks a lot
